docs/developer/decision-records/2022-07-01-get-claims/README.md

```diff
@@ -61,3 +61,44 @@ For each VerifiableCredential:
    10. From the issuer DID Document, participant B extracts the public key of the issuer and uses it to verify the JWS signature.  
    11. If the signature is successfully verified, it extracts the claims of the participant, each claim needs to be associated with its issuer.  
 12. The list of the claims will be used as input to apply the access policies.
+
+## Claims format
+
+The IdentityHubCredentialsVerifier returns a Map<String, Object>, it contains verifiable credentials with the following format:
+
+```json
+<verifiable-credential-id>: {
+  "vc": {
+    "id": <verifiable-credential-id>,
+    "credentialSubject": <claims>
```

Comment:
what is the format of `<claims>`?

Reply:
I added a quote to specify this:
"The credentialSubject field can contain claims with various format, see the [Credential Subject documentation](https://www.w3.org/TR/vc-data-model/#credential-subject) for more details."

Reply:
is it a JSON List? A JSON Object?

Reply:
@ouphi

Reply:
Given the doc it can be both, that's why I think it is better to just link to the documentation